meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/gdp-new-hmi.bb

```diff
@@ -1,5 +1,5 @@
 SRC_URI = "git://github.com/GENIVI/hmi-layout-gdp.git"
-SRCREV = "d0ca9d072012d8ac333537d9b922d9242e022efe"
+SRCREV = "79a4f69e7588a6efc4b4c95abec7794216588087"
```

Comment:
@zeenix 

My commit is not about side effect and 79a4f69e7588a6efc4b4c95abec7794216588087 is also for changing naming. So 79a4f69e7588a6efc4b4c95abec7794216588087 in component repo and my recipe change are for naming changing.

I thought that you don't understand this change.